conceptnet5/vectors/cli.py

```diff
@@ -300,12 +290,12 @@ def run_propagate(assoc_filename, embedding_filename, output_filename,
                       nshards=nshards, iterations=iterations)
 
 
-@cli.command(name='join_propagate')
+@cli.command(name='join_shard_files')
 @click.argument('filename', type=click.Path(writable=True, dir_okay=False))
 @click.option('--nshards', '-s', default=6)
 @click.option('--sort/--no-sort', default=True)
```

Comment:
Doesn't the default set to True make the sorting always happen?

Reply:
Nitpick: with `--sort` being an option now, could we call the short version of `nshards` `-n` instead of `-s`?

Reply:
Done.